app/org/sagebionetworks/bridge/dynamodb/DynamoScheduledActivityDao.java

```diff
@@ -57,48 +52,30 @@ public ScheduledActivity getActivity(String healthCode, String guid) {
         if (dbActivity == null) {
             throw new EntityNotFoundException(ScheduledActivity.class);
         }
+        dbActivity.setTimeZone(timeZone);
```

Comment:
The only place this was used before was to load an activity in order to update its start/end times. And in fact, that's all we're using it for now, as well. But it seemed more consistent to have it always set the timezone, rather than do it in one method and not the other, in case someone ever calls this besides the update code (where it's set to null because it doesn't matter).

Time zones are tested in ScheduledActivityServiceTest.retrievalActivitiesAcrossTimeAndTimeZones at length. I've added similar tests to DynamoScheduledActivityDaoMockTest for this method. It relies on the same code though (the ScheduledActivity implementation).